package.json

```diff
@@ -22,6 +23,7 @@
     "eslint": "^8",
     "eslint-config-next": "14.0.0",
     "postcss": "^8",
+    "prettier": "^3.0.3",
```

Comment:
btw, prettier is usually added as a dev dependency, we dont need it to appear in the final build https://prettier.io/docs/en/install.html